packages/cli/src/commands/logs/command.ts

```diff
@@ -1,9 +1,13 @@
 import { Command } from '../help';
 import { packageName } from '../../util/pkg-name';
 
+// has to be ms compliant
+// https://github.com/vercel/ms/blob/fe5338229cfdac6822891dcb9c24660b4d2e612b/src/index.ts#L95
+export const CommandTimeout = '5 minutes';
+
 export const logsCommand: Command = {
   name: 'logs',
-  description: 'Display logs for a specific deployment.',
+  description: `Display runtime logs for a specific deployment, if it is live, from now and for ${CommandTimeout} at most.`,
```

Comment:
**question**, since we are always following, wouldn't it be always live?

Reply:
Maybe the sentence was unclear. "being live" was about the deployement.
I can change to "if it is ready" maybe?

Reply:
hmm, are we trying to say that it will only display runtime logs if it is live?